README.md

```diff
@@ -1,83 +1,299 @@
+
+![fpcClientFlow](./fpcClientFlow.png)
+
+For more details about this, refer to the FPC transaction flow section in the FPC rfc document [here](https://github.com/hyperledger/fabric-rfcs/blob/main/text/0000-fabric-private-chaincode-1.0.md#fpc-transaction-flow).
+
+#### Client example by cc-tools-demo
+
+The following diagram is explaining the process where we modified the API server developed for demo on CC-tools ([CCAPI](https://github.com/hyperledger-labs/cc-tools-demo/tree/main/ccapi)) and modified it to communicate with FPC code.
+
+The transaction client invocation process, as illustrated in the diagram, consists of several key steps that require careful integration between FPC and CC Tools.
+
+1. Step 1-2: The API server are listening for reequests on a specified port over an HTTP channel and send it to the handler
+2. Step 3: The handler starts by determining the appropriate transaction invocation based on the requested endpoint and calling the corresponding chaincode API
+3. Step 4: The chaincode API is responsible for parsing and ensuring the payload is correctly parsed into a format that is FPC-friendly. This parsing step is crucial, as it prepares the data to meet FPC’s privacy and security requirements before it reaches the peer.
+4. Step 5: FPC utils is the step where the actual transaction invokation happens and it follows the steps explained in the previous diagram.
+
+![CCAPIFlow](./CCAPIFlow.png)
+
+## User experience
+
+(Marcus: Now we describe the flow or chaincode development (with cc-tools), the extra changes to compile it with FPC, do a test deployment with the fabric-samples test-network, and how to invoke a test transaction using the testing apiserver) ...
+
+We will describe from a user perspective a high overview on how to develop a chaincode using cc-tools that works with FPC achieving both ease of development and enhanced security of the chaincode.
+
+###### 1. Setting up the FPC dev environment
 
-![wrappingOrder](./wrappingOrder.png)
+First, any chaincode that is supposed to be working with FPC need some dependencies to develop and build the chaincode like: Ego, Protocol Buffers, SGX PSW, etc...
 
-Here's an example of how we did it for the chaincode on cc-tools-demo with chaincode as a service
+For this we should follow this section in FPC repo to setup the development environment [here](https://github.com/hyperledger/fabric-private-chaincode/tree/main?tab=readme-ov-file#getting-started).
+
+###### 2. Develop the chaincode in cc-tools and fpc
+
+Fortunately, since the stubwrapper for both cc-tools and fpc are implementing the same interface, the conversion to an fpc chaincode can be done by plug-and-play. This means, the user should start by developing the chaincode using cc-tools and at the main loop where he passes the chaincode instance to the server to start it, he needs to wrap it with `fpc.NewPrivateChaincode`. For example, have a look at the cc-tools-demo chaincode below
```

Comment:
change instead of referring to the user as HE to refer to them as THEY ... so they pass, the need, etc

Reply:
Done